preprod.healdata.org/metadata/aggregate_config.json

```diff
@@ -529,8 +529,572 @@
 					"authz": "/programs/open",
 					"study_url": "https://pdaps.org/datasets/laws-regulating-administration-of-naloxone-1501695139",
 					"dataset_1_title": "Naloxone_Data_09112020.xlsx, 20180810_Good_Samaritan_Law_Stat_Data.xlsx, 20170216-RM-Stat-Data.xlsx"
-				}
-			}
-		}
-	}
-}
+				},
+                "57b45d83d6c9e7e8693ccdfb": {
+                    "__manifest": [
+                        {
+                            "md5sum": "43a9ac4c3bd4e38aede09f7d74b8d1a3",
+                            "file_size": 210648,
+                            "file_name": "",
```

Comment:
Could you provide the file_names? Thanks much!!

Reply:
Hey Xenia -- @jawadqur  it looks like there are no file names indexed (ie they are blank). For example, here's the manifest -- does this not matter and should I add the AWS file names (with subdirectories)?
[manifest_externaldata.csv](https://github.com/uc-cdis/cdis-manifest/files/7899989/manifest_externaldata.csv)

Reply:
You would find the name in the `urls` columns (after the last `/`).

Reply:
@xritter1  thanks! Ok I added the file names